go.mod

```diff
@@ -22,4 +22,6 @@ require (
 	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776 // indirect
 )
 
+replace github.com/buildpacks/imgutil => github.com/jjbustamante/imgutil v0.0.0-20210810205516-1924767685e5
+
```

Comment:
Let's make sure that this doesn't make it in the final merge 🤓

Reply:
Yeah! I will remove that once the imgUtil PR has been merged

Reply:
Removed in the following [commit](https://github.com/buildpacks/lifecycle/pull/685/commits/7ce8bb8297e15af0aab609e3df4c462172972ef9)